### PR TITLE
Move note encryption types around

### PR DESCRIPTION
**Pull request by dconnolly**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -58,9 +58,12 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: tarpaulin
-          args: --verbose --count --workspace --timeout 180 --out Html
+          args: --verbose --count --workspace --timeout 300 --out Xml
```
Comment (hdevalence) [inline .github/workflows/main.yml:61]: Just curious -- what's the motivation for switching to `Xml`?

File: zebra-chain/src/transaction/joinsplit.rs [hunk 2]
```diff
@@ -54,7 +46,7 @@ pub struct JoinSplit<P: ZkSnarkProof> {
     /// [`Bctv14Proof`](crate::proofs::Bctv14Proof).
     pub zkproof: P,
     /// A ciphertext component for this output note.
-    pub enc_ciphertexts: [EncryptedCiphertext; 2],
+    pub enc_ciphertexts: [sprout::EncryptedCiphertext; 2],
```
Comment (hdevalence) [inline zebra-chain/src/transaction/joinsplit.rs:49]: this is a nice touch

File: .github/workflows/main.yml (hunk 1 above)
Comment (dconnolly) [reply .github/workflows/main.yml:61]: So they could be input to CodeCov which makes the nice inline comment on the PR/has a web UI. I tried to output both xml and html but I think I needed a comment between args.

Comment (hdevalence) [reply .github/workflows/main.yml:61]: Cool! I see the graphic now, it looks great

**Pull request merged**